docs/source/daml-integration-kit/index.rst

```diff
@@ -96,8 +96,25 @@ GA (`tracking GitHub issue <https://github.com/digital-asset/daml/issues/661>`__
 DAML Ledgers built or in development
 ====================================
 
-**TODO (ALPHA):** explain what ledgers exist; and what state they are in.
-(`GitHub issue <https://github.com/digital-asset/daml/issues/673>`__)
+The following table lists the ledgers that support DAML, or are implementing
+support for running DAML.
+
++-----------------------------------------------+------------------+----------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------+
```

Comment:
Looks fine, but I suggest using a list table instead https://sublime-and-sphinx-guide.readthedocs.io/en/latest/tables.html